Hearthstone Deck Tracker/Hearthstone/Watchers.cs

```diff
@@ -20,18 +20,21 @@ static Watchers()
 			PackWatcher.NewPackEventHandler += (sender, args) => PackUploader.UploadPack(args.PackId, args.Cards);
 			DungeonRunWatcher.DungeonRunMatchStarted += DeckManager.DungeonRunMatchStarted;
 			DungeonRunWatcher.DungeonInfoChanged += DeckManager.UpdateDungeonRunDeck;
+			FriendlyChallengeDialogVisibilityWatcher.OnDialogVisibilityChanged += (sender, args) => { if (args.DialogVisible && Config.Instance.FlashHsOnFriendlyChallenge) User32.FlashHs(); };
```

Comment:
No one line if statements please, this being multi line or a method is fine.

Reply:
I'd like to create a method for this, not sure where to put it though.

Reply:
You can just add a static method in the Watchers class.